CONTRIBUTING.md

```diff
@@ -577,14 +577,18 @@ The following general guidelines apply in every language if not noted otherwise
    foo # This is one group of code.
 
    bar # This is another group of code which is slightly related to `foo`.
-
-
+   
+   
    baz # This is a group of code which is neither related to `foo` nor `bar`.
+   
    ```
-
+   
    Larger groups of code should be set apart using comments like the following.
 
-   ```bash
+   ```
+
+   
+   ​```bash
```

Comment:
These are the changes I've meant in my [previous comment](https://github.com/FreeProving/guidelines/pull/1#discussion_r431032124) which should be undone. The closing ` ``` ` has no matching opening ` ```bash ` and is thus messing with GitHub's rendering of the code block.

````suggestion
   ```bash
````